installation-names/README.md

```diff
@@ -0,0 +1,33 @@
+# Assigning installation names
+
+The purpose of this RFC is to specify how we select installation names.
+
+## High level process
+
+- A Giant Swarm team is responsible for providing name candidates in a dedicated, public git repository.
+
+- Customers, when preparing for a new installation, select a name from the repository. A pull request ensures that the claimed name is removed from the above repository.
+
+## Providing name candidates
+
+When providing name candidates, we apply the following criteria:
+
+- ASCII characters a-z only
+- Length minimum 5, maximum 8 characters
+- Must not have been in use for an installation previously
+- Should be easy to pronounce, to spell, and to type in an english language context
```

Comment:
`Should be easy to pronounce, to spell` might be very different across different countries, a word that might sound and spell easily for you might not work for someone else

the current list in the installation-names repo is kinda opinionated and I don't like several names

Reply:
I tried to make it more precise by adding "in an english language context".

If you can tell _why_ you don't like several names, we can try to derive criteria from that. If you simply don't like some names, please provide a PR to the installation-names repo.

Reply:
I'd say that's something we can try to accomplish in the installation-names repo, however it will be hard to avoid completely.